plugins/org.eclipse.fordiac.ide.gef/src/org/eclipse/fordiac/ide/gef/tools/AdvancedPanningSelectionTool.java

```diff
@@ -134,8 +134,7 @@ protected boolean handleButtonUp(final int which) {
 	protected boolean handleDrag() {
 		if (isInState(PAN_IN_PROGRESS) && (getCurrentViewer().getControl() instanceof FigureCanvas)) {
 			final FigureCanvas canvas = (FigureCanvas) getCurrentViewer().getControl();
-			// canvas.scrollTo(viewLocation.x - getDragMoveDelta().width, viewLocation.y -
-			// getDragMoveDelta().height);
+			canvas.scrollTo(viewLocation.x - getDragMoveDelta().width, viewLocation.y - getDragMoveDelta().height);
```

Comment:
this should not be renabled as this is handled by the expose helper added on one of the commits of this branch.